src/main/java/com/purbon/kafka/topology/roles/CCloudAclsProvider.java

```diff
@@ -79,40 +80,37 @@ private TopologyAclBinding translateIfNecessary(TopologyAclBinding binding) thro
             + "Service Account names into ID(s). At some point in time this will not be required anymore, "
             + "so you can configure this out by using ccloud.service_account.translation.enabled=false (true by default)");
 
-    Principal principal = Principal.fromString(binding.getPrincipal());
-    long numericServiceAccountId = serviceAccountIdByNameMap.getOrDefault(principal.getServiceAccountName(), SERVICE_ACCOUNT_NOT_FOUND);
-    if (numericServiceAccountId == SERVICE_ACCOUNT_NOT_FOUND) { // Translation failed, so we can't continue
+    ConfluentCloudPrincipal principal = ConfluentCloudPrincipal.fromString(binding.getPrincipal());
+    long numericServiceAccountId =
```

Comment:
nit-pick: I think the formatting has changed, but the code has not. So, I'd prefer to restore the original formatting, otherwise it just shows up as another (distracting?) change

Reply:
This should be okay as this was modified by us in older PR, I think it wasn't formatted then. So, for the `julie` master branch it's anyway shown as a change